src/ontology/mondo-ingest.Makefile

```diff
@@ -486,18 +486,10 @@ slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/%_map
 slurp-%: slurp/%.tsv
 	@echo "$@ completed".
 
-.PHONY: slurp-no-updates-%
-slurp-no-updates-%: slurp/%.tsv
-	@echo "$@ completed".
-
 .PHONY: slurp-docs
 slurp-docs:
 	python3 $(SCRIPTSDIR)/migrate.py --docs
 
-.PHONY: slurp-all-no-updates
-slurp-all-no-updates: $(foreach n,$(ALL_COMPONENT_IDS), slurp-no-updates-$(n))
-	@echo "$@ ($^) completed".
```

Comment:
## Deletion of unused goals
Deletes:
- `slurp-all-no-updates`
- `slurp-no-updates-%`

Thes currently do the exact same thing as `slurp-all` and `slurp-%`. They're not called by any other goals, so they're useless.

I am trying to remember what these goals were originally for.

It may be that they were a variation of running the slurp that did not involve doing some `pip install`s beforehand. The `pip install` would have been because ODK did not have high enough versions of some libraries at the time, likely OAK or maybe `sssom-py`.

I also have some recollection of `-B` being part of / necessary for some goal, maybe for part of the `slurp` pipeline, and this `-no-updates` variation would have been the same thing, but without `-B`.

Either way, whatever purpose they had appears to no longer be valid. They're doing nothing here, so I'm deleting them.